lib/slack.py

```diff
@@ -104,30 +111,51 @@ def __build_message__(self, suggested_video, channel, username, search_id, index
 
         return params
 
+    def __get_videos__(self, search_id):
+        self._mysql.execute("USE tubey;")
+        self._mysql.execute("select videos from video_suggestions where search_id = {}".format(search_id))
+        videos = literal_eval(self._mysql.fetchone()[0])
+
+        return videos
+
     def suggest_video(self, user_info, team_info, channel_info, query="", action_info={}):
         # Sends a video suggestion in an ephemeral message
         if 'name' in action_info.keys() and action_info['name'] == 'shuffle':
             button_value = loads(action_info['value'])
             search_id = button_value['search_id']
             channel = channel_info['id']
             username = user_info['name']
```

Comment:
If you're only going to use a variable once, it doesn't need to be one (unless it makes something more clear, but this is already clear). Also I thought we were passing user_id to build message but apparently it's username?